### PR TITLE
Create parse_mito_blast.xml

### PR DESCRIPTION
**Pull request by Nadolina**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/parse_mito_blast.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+ <tool id="parse_mito_blast" name="Parse mitochondrial blast" version="1.0" python_template_version="3.5">
+    <description>overlap-conscious coverage calculation between scaffold-accession number pairs from blast output</description>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION">parse_mito_blast</requirement>
+        <requirement type="package" version="@TOOL_VERSION@">pandas</requirement>
+        <requirement type="package" version="@TOOL_VERSION@">argparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[python3 parse_mito_blast --blastout BLAST_RESULTS --covreport OUTPUT_FILE_NAME]]>
```
Comment (bgruening) [inline tools/parse_mito_blast.xml:8]:
```suggestion
    <command detect_errors="exit_code"><![CDATA[
parse_mito_blast 
    --blastout $blastout 
    --covreport
    $output
]]>
``

File: tools/parse_mito_blast.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+ <tool id="parse_mito_blast" name="Parse mitochondrial blast" version="1.0" python_template_version="3.5">
+    <description>overlap-conscious coverage calculation between scaffold-accession number pairs from blast output</description>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION">parse_mito_blast</requirement>
+        <requirement type="package" version="@TOOL_VERSION@">pandas</requirement>
```
[the remark this answers is not in the document]
Comment (Nadolina) [reply tools/parse_mito_blast.xml:5]: Got it thanks!

File: tools/parse_mito_blast.xml (hunk 1 above)
Comment (Nadolina) [reply tools/parse_mito_blast.xml:8]: Those two tick marks at the end are intentional?

Comment (bgruening) [reply tools/parse_mito_blast.xml:8]: No, removed :)

**Review by bernt-matthias — COMMENTED** [review]
Please add a `.shed.yml` file.

File: tools/parse_mito_blast.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <requirement type="package" version="@TOOL_VERSION@">argparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[python3 parse_mito_blast --blastout BLAST_RESULTS --covreport OUTPUT_FILE_NAME]]>
+    </command>
+    <inputs>
+        <param name="BLAST_RESULTS" label="TSV generated by mito-blast" type="data"/>
+        <param name="OUTPUT_FILE_NAME" label="Name to generate for report" type="text"/>
+    </inputs>
+    <outputs>
+        <data name="OUTPUT_FILE" label="${tool.name} on ${on_string}" format="csv"/>
```
[the remark this answers is not in the document]
Comment (Nadolina) [reply tools/parse_mito_blast.xml:22]: If I have two outputs now should they both be named output ? Does the name matter?

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/parse_mito_blast.xml:22]: Yes, the name you reference in your command section. So please use a meaningful other name.

[the remark this answers is not in the document]
Comment (Nadolina) [reply tools/parse_mito_blast.xml:22]: But what if my script doesn't have a --output flag like with Bellerophon; I initially had variables to allow for more specific names, but now just have hardcoded names for each of the output files and I've defined two outputs in the xml with those names, in that instance they don't need to be referenced in the command?

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/parse_mito_blast.xml:22]: You still need to have different names, but you can use from_work_dir="my_outputname.txt" 
https://docs.galaxyproject.org/en/latest/dev/schema.html#id77

**Review by bgruening — APPROVED** [review]

**Pull request merged**